### ISSUE TITLE
Multiple audio player instances

### ISSUE THREAD

**Issue by randomboygh** [Selz/plyr#475]
- [x] Issue does not already exist
- [ ] Issue observed on https://plyr.io

### Expected behaviour 
One audio player for each audio object

### Actual behaviour
Each audio object have a multiple audio player instances. But video player works fine

### Environment
- Browser: Firefox
- Version:  50.1.0
- Operating System: Windows 7
- Version: 6.1.7601

### Steps to reproduce 
I want to add plyr on my social network, but I have a trouble.
Write this code for post files:
```html
    <div class="post-files">
        <?php foreach ($files as $file) : ?>
                <?php if ($file->getMimeBaseType() == "audio") : ?>
                    <audio controls>
                        <source src="<?php echo $file->getUrl(); ?>">
                    </audio>
                <?php elseif ($file->getMimeBaseType() == "video") : ?>
                    <video controls>
                        <source src="<?php echo $file->getUrl(); ?>">
                    </video>
                <?php endif; ?>
        <?php endforeach; ?>
        <script>
            plyr.setup({iconUrl: "<?php echo $this->theme->getBaseUrl() . '/img/plyr.svg'; ?>"});
            plyr.loadSprite("<?php echo $this->theme->getBaseUrl() . '/img/plyr.svg'; ?>");
        </script>
    </div>
```
It works fine without plyr.js, but with the one audio players have a copy for each audio object in a post and each audio object on a page.
So, on attached pics you can see, what I mean. First with plyr.js, second without plyr.js:
1
![1](http://storage3.static.itmages.ru/i/17/0108/h_1483895279_3110745_9dc39fe2e7.png)
2
![2](http://storage3.static.itmages.ru/i/17/0108/h_1483895279_3273273_5bc5d1cdc1.png)

I have a problem with my code or plyr.js have a bug?

*Sorry for my english*

### Relevant links

**Comment by randomboygh** [comment]
I tried to solve the problem.
Every run plyr.setup() I've got an another audio player instance, but only audio, not video. It's strange.
![](http://storage1.static.itmages.ru/i/17/0109/h_1483926881_9391048_2be1255d02.png)

**Comment by SamPotts** [comment]
Yep someone else has raised this. Will take a look 👍

**Comment by Blucreation** [comment]
This going to be fixed any time soon?

**Comment by sampotts** [comment]
Yeah, I'm working on it with the new version. Can't give a firm ETA though.

**Comment by Galkon** [comment]
Can this get fixed please?? I was so excited to be using plyr, but this completely screws me. Interestingly enough - it only happens on autoplay. Which is pretty crucial for my purpose.

**Comment by rftb** [comment]
@sampotts Any news on an ETA for the new version with this fixed? Suuuuper important! Thanks! Love the customization possibilities with your player, awesome job.

**Comment by JoeEarly** [comment]
Have this issue now too for some reason. Affecting video files for me, standard mp4s

**Comment by sampotts** [comment]
Closing this one as I think it's fixed but feel free to re-open if not solved in v3 👍
